aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -12,3 +12,93 @@ proposals:
       - Framework:
           bytecode_version: 6
           git_hash: ~
+  - name: change_operator_beneficiary
+    metadata:
+      title: "Changing beneficiaries for operators"
+      description: "AIP-51: This allows operator to change their beneficiaries."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/251"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - OPERATOR_BENEFICIARY
```

Comment:
Are these names correct?

Reply:
These are the constant names used in the code. @perryjrandall , what are the proper names to put in? In other words, where should these names come from?

Reply:
Might not matter much but we usually do all lower case here

Reply:
Done making them lowercase.